README.md

```diff
@@ -524,7 +524,7 @@ class SessionsController < ApplicationController
 
   def destroy
     logout
-    redirect_to root_path, notice: "Singed out."
+    redirect_to root_path, notice: "Signed out."
```

Comment:
Good catch! Are you also able to update this in [the controller](https://github.com/stevepolitodesign/rails-authentication-from-scratch/blob/c9de70f76cbae7391c18c73d0c66b0f6bdeb65c4/app/controllers/sessions_controller.rb#L14)?

Reply:
Updated the controller.